### PR TITLE
server _auth and _authAdmin validation

### PR DESCRIPTION
**Pull request by tri-et**

### CODE REVIEW

File: server/util.js [hunk 1]
```diff
@@ -2,4 +2,16 @@ export const _auth = authCustomer => {
...
     throw new Error('Please login first!')
   }
+  if (authCustomer.roles) {
+    throw new Error('Account not customer!')
+  }
+}
+
+export const _authAdmin = authAdmin => {
+  if (!authAdmin) {
+    throw new Error('Please login first!')
```
Comment (hemvanh) [inline server/util.js]: Please login as an Admin first!

File: server/util.js [hunk 2]
```diff
@@ -2,4 +2,16 @@ export const _auth = authCustomer => {
...
+  }
+  if (authAdmin.roles == undefined) {
+    throw new Error('Account not admin!')
```
Comment (hemvanh) [inline server/util.js]: This Account is not an Admin!

File: server/util.js [hunk 3]
```diff
@@ -2,4 +2,16 @@ export const _auth = authCustomer => {
   if (!authCustomer) {
     throw new Error('Please login first!')
   }
+  if (authCustomer.roles) {
+    throw new Error('Account not customer!')
```
Comment (hemvanh) [inline server/util.js]: This is not a Valid Customer account!

File: src/store/customer/actions.js [hunk 4]
```diff
@@ -136,7 +136,8 @@ export const fetchCustomer = ({commit}) => {
     }
   }`)
     .then(({data}) => {
-      commit('setCustomer', data.getCustomer)
+      if (data.errors) _alert(data.errors[0].message, 'warning')
```
Comment (hemvanh) [inline src/store/customer/actions.js:139]: should the error already been taken care in: .catch(err => { ... ?

**Review by hemvanh — CHANGES_REQUESTED** [review]

File: src/store/customer/actions.js (hunk 4 above)
Comment (tri-et) [reply src/store/customer/actions.js:139]: catch just show error api graphql don't show error customize

**Review by hemvanh — APPROVED** [review]

**Pull request merged**